docker/kubernetes-agent-tentacle/Dockerfile

```diff
@@ -1,3 +1,6 @@
+ARG RuntimeDepsTag
```

Comment:
This needs to be defined at the very top as a 'global' ARG because the Dockerfile uses a multi-stage build. Each `FROM` statement declares a variable scope which lasts until either the next FROM or the end of file. This means that putting this line right above the FROM line where it is used will cause it to fall under the previous FROM's scope and hence not work.